src/controllers/albumController.js

```diff
@@ -33,6 +69,26 @@ const updateAlbum = async (req, res) => {
     throw new CustomError.NotFoundError(`No album with id ${albumId}`);
   }
   res.status(StatusCodes.OK).json({ album });
+  /*
+     #swagger.summary = 'Fetch an album  by id and update it'
+     #swagger.parameters['id'] = {
+        description: 'Mongo ObjectID of the album to fetch',
+     }
+          #swagger.parameters['album properties to update'] = {
+      in: 'body',
+      description: 'Album information to use for update',
+      required: true,
+      type: 'object',
+      schema: { $ref: '#/definitions/NewAlbum' }
+     }
+     #swagger.responses[200] = {
+				description: 'Album successfully fetched and updated.',
+				schema: { $ref: '#/definitions/Album' }
```

Comment:
We return the response wrappd in an object with the key `"album"`

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/c5d40f49-dc9e-4fe0-b0b2-ff263b1c39f9)

so this should be 

```suggestion
				schema: { album: { $ref: '#/definitions/Album' } }
```